yoti-sdk-api/src/main/java/com/yoti/api/client/identity/constraint/SourceConstraint.java

```diff
@@ -0,0 +1,76 @@
+package com.yoti.api.client.identity.constraint;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import com.yoti.api.client.identity.policy.WantedAnchor;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class SourceConstraint implements Constraint {
+
+    private static final String TYPE = "SOURCE";
```

Comment:
(nit) since this is only used once maybe inline it? Just a thought - no biggie if you wanna keep it as a constant.